hail/src/main/scala/is/hail/utils/richUtils/RichCodeRegion.scala

```diff
@@ -14,6 +14,8 @@ class RichCodeRegion(val region: Code[Region]) extends AnyVal {
   def trackAndIncrementReferenceCountOf(other: Code[Region]): Code[Unit] =
     region.invoke[Region, Unit]("addReferenceTo", other)
 
+  def takeOwnershipOfAndClear(other: Code[Region]): Code[Unit] = region.invoke[Region, Unit]("move", other)
```

Comment:
I think this is backwards

Reply:
`Region.move` is "move to"

Reply:
ugh, thanks. I said it correctly in my head when I copy/pasted the line above, and then forgot to change it 🤦